CONTRIBUTING.md

```diff
@@ -1,139 +1,138 @@
+3. If your proposed fix is not in the [open issues backlog](https://github.com/uswds/uswds/issues), [create an issue](https://github.com/uswds/uswds/issues/new?assignees=&labels=type%3A+feature+request&template=feature_request.md&title=) describing your proposal. This doesn’t mean we don’t want you to create a pull request. We simply want to start the process with an online conversation first. Plus, other community members might have supporting thoughts to add to your proposal. If you’ve already got a pull request, no worries. Go ahead and attach it to the issue. 
+
+
+### ✨ Proposing something else?
+
+If you’d like to contribute something else that doesn’t fall into any of the above, we’d still love to hear about it. Just create an issue or a discussion, and we can talk about it.  
+
+## How we prioritize
+
+Once you’ve submitted a contribution, we triage them based on the following considerations:
+1. Size: Can we accomplish this in a sprint or will this take longer? 
+2. Severity: What type of functionality is impacted? Is there a workaround?
+3. Priority: Does this align with our vision and roadmap goals? 
+
+Note: We prioritize issues impacting accessibility. 
+
+These considerations help us decide if and when we can work on the issue. If the issue is accepted, we scheduled them for an upcoming sprint (a 2-week work period).
+
+You can stay up to date on the status of your contributions through [GitHub email notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications) (external link) and the assigned labels on the issue. 
+
+## Common terms
+
+There can be a lot of jargon when discussing how you can contribute to the Design System. We’ve included some common terms we use below.
+
+- **Backlog** - list of deliverables (like a feature request, enhancement, or bug) that should be implemented into upcoming product development. 
+- **Bug** - problem resulting in something not working properly or as expected.
+- **Contribution** - when a community member gives back in a way that enhances the Design System by proposing a new idea, enhancement, or fix that’s released through the system for other people to use.
+- **Enhancement** - a proposal to make something existing in the Design System work better. 
+- **Feature request** - a proposal for something new to be included to the Design System. 
+- **Fork** - a copy of a repository that you manage. 
+- **Open source** - something that can be viewed, modified, and shared by anyone in the public with permissions enforced through an open source license.
+- **Pull request** - a way to notify project team members when a contributor/ developer wants to merge new code changes into a main project repository. You can read more on [GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) (external link).
```

Comment:
Extra space; removed external link reference

```suggestion
- **Pull request** - a way to notify project team members when a contributor/developer wants to merge new code changes into a main project repository. You can read more on [GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests).
```

Reply:
I fixed the extra space - thank you!